projects/ngx-stripe/src/lib/interfaces/payment-intent.ts

```diff
@@ -135,3 +136,8 @@ export interface PaymentMethodResult {
   metadata: { [key: string]: any };
   type: string;
 }
+
+export interface PaymentMethodResult {
```

Comment:
This needs to be in a wrapper, as the promise always returns. See https://stripe.com/docs/stripe-js/reference#stripe-create-payment-method